config/peerpods/podvm/lib.sh

```diff
@@ -197,7 +197,12 @@ function prepare_source_code() {
     fi
 
     # links must be relative
-    if [[ "$CONFIDENTIAL_COMPUTE_ENABLED" == "yes" ]]; then
+    if [[ -e /tmp/kata-opa/custom.rego ]] ; then
```

Comment:
How about adding a folder in podvm with sample policy file and a README describing the details on how policy is handled in OSC (via the configmap etc) along with examples on using the custom policy  as well as another example of setting policy via the annotation?